web-ui/src/components/member-directory/AdminMemberCard.jsx

```diff
@@ -210,6 +215,11 @@ const AdminMemberCard = ({ member, index }) => {
               open={open}
               onClose={handleClose}
               onSave={async member => {
+                let resGetMember = await getMember(member.id, csrf);
+                let oldMember =
+                    resGetMember.payload && resGetMember.payload.data && !resGetMember.error
```

Comment:
`resGetMember.payload>.data && !resGetMember.error`

Reply:
Added. Thanks for the comments @mvolkmann.